modules/fishhook/1.0/config/default.yaml

```diff
@@ -0,0 +1,38 @@
+lcr-modules:
+    
+    fishhook:
+        inputs:
+            # Available wildcards: {seq_type}
+            master_maf: "__UPDATE__"
+            sample_sets: "__UPDATE__"
+
+        sample_set: ["__UPDATE__"]
+        seq_types: ["genome", "capture"]
+        include_non_coding: True
+        prepare_mafs: "{SCRIPTSDIR}/generate_smg_inputs/1.0/generate_smg_inputs.R"
+        src_dir: "{MODSDIR}/src/"
+
+        options:
+            tiles_size: 1000 #This is the size of tiles you want to split the genome into
```

Comment:
This is assuming the tiling will also be used - but there is also a way to run it without tiles, right? Just wondering if the documentation here should be something like the one below that says Leave blank ...

Reply:
Done, using tile will be the default mode, is tile_size is blank, user should include a gene list for gene-centric analysis.